src/components/home/sprint-view-card-content/user-sprint-view-card.tsx

```diff
@@ -0,0 +1,118 @@
+  const [timeEntries, setTimeEntries] = useState<number[]>([]);
+  const { allocationsApi, projectsApi, timeEntriesApi } = useLambdasApi();
+  const setError = useSetAtom(errorAtom);
+
+  useEffect(() => {
+    getAllocationsAndProjects();
+  }, [loggedInPerson]);
+
+  /**
+   * Get allocations, projects names, colors and time entries
+   */
+  const getAllocationsAndProjects = async () => {
+    setLoading(true);
+    if (loggedInPerson) {
+      try {
+        let fetchedAllocations = await allocationsApi.listAllocations({
+          personId: loggedInPerson.id.toString(),
+          startDate: new Date(),
+          endDate: new Date()
+        });
+        const fetchedProjects = await projectsApi.listProjects();
+        const {filteredallocations, filteredProjects} = filterAllocationsAndProjects(fetchedAllocations, fetchedProjects);
+        const totalTimeEntries = await Promise.all(filteredallocations.map(async (allocation) => {
+          try {
+            if (allocation.project) {
+              const fetchedTimeEntries = await timeEntriesApi.listProjectTimeEntries({
+                projectId: allocation.project,
+                startDate: allocation.startDate,
+                endDate: allocation.endDate
+              });
+              let totalMinutes = 0;
+              fetchedTimeEntries.forEach((timeEntry: TimeEntries) => {
+                if (loggedInPerson && timeEntry.person === loggedInPerson.id) {
+                  totalMinutes += (timeEntry.timeRegistered || 0);
+                }
+              });
+              return totalMinutes;
+            }
+          } catch (error) {
+            setError(`${strings.sprintRequestError.fetchTimeEntriesError}, ${error}`);
+          }
+          return 0;
+        }));
+        setProjects(filteredProjects);
+        setAllocations(filteredallocations);
+        setTimeEntries(totalTimeEntries);
+      } catch (error) {
+        setError(`${strings.sprintRequestError.fetchError}, ${error}`);
+      }
+    }
+    setLoading(false);
+  }
+
+  /**
+   * Combines allocations and projects data for chart
+   */
+  const createChartData = (): SprintViewChartData[] => {
+    return allocations.map((allocation, index) => {
+      return {
+        projectName: projects[index].name || "",
+        timeAllocated: totalAllocations(allocation),
+        timeEntries: timeEntries[index],
+        color: projects[index].color || ""
+      };
+    })
+  }
+
+  /**
+   * Renders sprint view bar chart
+   */
+  const renderBarChart = () =>
+    <>
+      {allocations.length ?
+        <CardContent sx={{ display: "flex", justifyContent: "left" }}>
+          <SprintViewBarChart chartData={createChartData()} />
+        </CardContent>
+        :
+        <Typography style={{ paddingLeft: "0" }}>{strings.sprint.noAllocations}</Typography>
+      }
+    </>
+
+  return (
+    <>
+      {!loggedInPerson || loading ?
+        <Skeleton /> : renderBarChart()
+      }
+    </>
```

Comment:
indentation could be improved, if biome is running then saving this file should fix it